packages/react-accordion/src/components/AccordionItem/useAccordionItem.ts

```diff
@@ -4,27 +4,26 @@ import { useTabsterAttributes } from '@fluentui/react-tabster';
 import { useContextSelector } from '@fluentui/react-context-selector';
 import { AccordionContext } from '../Accordion/AccordionContext';
 import { AccordionToggleEvent } from '../Accordion/Accordion.types';
+import { resolveShorthand } from '@fluentui/react-utilities';
 
 /**
  * Const listing which props are shorthand props.
  */
-export const accordionItemShorthandProps: Array<keyof AccordionItemSlots> = [];
+export const accordionItemShorthandProps: Array<keyof AccordionItemSlots> = ['root'];
 
 /**
  * Returns the props and state required to render the component
  * @param props - AccordionItem properties
  * @param ref - reference to root HTMLElement of AccordionItem
  */
 export const useAccordionItem = (
-  { value, ...props }: AccordionItemProps,
+  { value, disabled = false, ...props }: AccordionItemProps,
```

Comment:
```suggestion
  props: AccordionItemProps,
```

nit: Let's move destruction into function body to make it more readable